app/index.js

```diff
@@ -433,6 +433,7 @@ const initialState = {
 };
 
 const fontConfig = {
+  // TODO: specify all fonts in fonts.less here!
```

Comment:
@tiagoalvesdulce @vctt94 @alexlyp 
I just noticed that we should migrate `fonts.less` content and provide all fonts here as well, and thought to do include it to this PR as well.
For that one small change is required in `pi-ui` side, currently, this is the only way we can define fonts with pi-ui, apparently this isn't enough to specify all fonts needed for decrediton: 
```
// fonts.less
@font-face {
  font-family: 'Inconsolata';
  font-weight: 700;
  src: url('@{font-root}/Inconsolata-Bold.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Inconsolata';
  font-weight: 400;
  src: url('@{font-root}/Inconsolata-Regular.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro';
  font-weight: 300;
  src: url('@{font-root}/SourceSansPro-Light.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro';
  font-weight: 300;
  font-style: italic;
  src: url('@{font-root}/SourceSansPro-LightItalic.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro';
  src: url('@{font-root}/SourceSansPro-Regular.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro';
  font-style: italic;
  src: url('@{font-root}/SourceSansPro-Italic.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro';
  font-weight: 700;
  src: url('@{font-root}/SourceSansPro-Bold.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro';
  font-weight: 700;
  font-style: italic;
  src: url('@{font-root}/SourceSansPro-BoldItalic.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro Semi Bold';
  font-weight: 600;
  src: url('@{font-root}/SourceSansPro-SemiBold.ttf'); /* stylelint-disable-line */
}

@font-face {
  font-family: 'Source Sans Pro';
  font-weight: 600;
  font-style: italic;
  src: url('@{font-root}/SourceSansPro-SemiBoldItalic.ttf'); /* stylelint-disable-line */
}
``` 

I can extend pi-ui to accept array of font-face definitions instead of object to support such set of fonts, which gonna lead to a small change in `pigui` as well, but I can take care of changes in `pi-ui` & `pigui` if the approach make sense to you guys, what do you think ?

Reply:
So, I've adjusted `pi-ui`'s `ThemeProvider` to accept fonts array & updaated pigui code base to provide array of fonts here:
https://github.com/decred/politeiagui/pull/1897 & https://github.com/decred/pi-ui/pull/254

Now I would be able to provide all the fonts in `fonts.less` and get red of it as well.

:no_good:

Reply:
Done!